community/modules/compute/htcondor-execute-point/variables.tf

```diff
@@ -185,3 +185,13 @@ variable "guest_accelerator" {
     error_message = "The HTCondor module supports 0 or 1 models of accelerator card on each execute point"
   }
 }
+
+variable "name_prefix" {
+  description = "Name prefix given to hostnames in this group of execute points; must be unique across all instances of this module"
```

Comment:
Just a question, not requesting any action in this PR: is there a way to validate that the variable value is unique across all instances of this module?

Reply:
Not in a good way, and also not without adding some functionality to the type of modules that we can write.